### ISSUE TITLE
Episodes in show list are missing the watched mark

### ISSUE THREAD

**Issue by ThibaultNocchi** [jellyfin/jellyfin-vue#992]
**Describe The Bug**
In a show page, the episodes are missing their watched mark.

**Screenshots**
![print_screen_2021-04-10-15-22-31](https://user-images.githubusercontent.com/1619359/114271214-9717b680-9a10-11eb-9181-b10a6b8ad2bc.png)
![print_screen_2021-04-10-15-22-22](https://user-images.githubusercontent.com/1619359/114271217-98e17a00-9a10-11eb-974c-cd3adfd9cd18.png)

**Comment by KeyboardDabbler** [comment]
I think this is more of a discussion to be had rather than a bug. How will the user know that season 1 is complete without first scrolling the page to check every episode. The use of season "tabs" is clean but limited. I added screens from my working jellfin-web to compare.

![Screenshot (10)](https://user-images.githubusercontent.com/53971316/119069401-2fef1800-ba3a-11eb-836b-ff57041b3b4e.png)
![Screenshot (13)](https://user-images.githubusercontent.com/53971316/119069408-341b3580-ba3a-11eb-9812-b8108890840a.png)
![Screenshot (14)](https://user-images.githubusercontent.com/53971316/119069415-37162600-ba3a-11eb-8326-1a6bf444d704.png)

**Comment by ThibaultNocchi** [comment]
This is actually a good point I didn't think of. I gotta look a little bit into that. Though that doesn't remove the need to have the tick on single episodes too :p

**Comment by Perseusdehond** [comment]
Thinking of a possible solution for what KeyboardDabbler said, I can across this: https://vimeo.com/399137601 
It shows a redesign of HBO Max, but I'm interested in the 00:08 and 00:11 times.

At the 00:08 point you can see that when you click on a serie, you would see all your seasons at the left, including if you've already completed a season or not. And then you see a just compact cards for the episodes without much description and just a title and episode number. - I think that this would be a good solution for going to the episode and season you want and seeing what you've already watched. (That card gridview also doesn't show any spoilers as there is no description, which could be a plus for some).

At the 00:11 point you can see that when you click on an episode, the column on the right changes from the seasons to the episodes. This would allow for quick switching between them. It also shows the full description of the episode on the right where you could change audio, subtitle tracks etc. or look at some actors.

I think that this would look good if we would change it to something that fits into the design of Jellyfin Vue. But it would certainly work with a tabbed Disney+ like design. As you have tabs on top with actors, images, information and then also **episodes** (or however we want to call it). And then when you click on the episodes tab, you would see a design sort of like in the video above.

Maybe this could be a solution for showing seasons and episodes with a tabbed design while also improving the speed of certain actions (at 00:11 it looks a bit like my proposed cast/crew redesign with the sidemenu for quick switching)?

**Comment by ThibaultNocchi** [comment]
That's an interesting take. I personally find the transitions not that great, but those distinct pages can be done. I don't have hard feelings against the current or this way, so I'll just report to @MrTimscampi and @ferferga :p

**Comment by Perseusdehond** [comment]
I don't think we should change the transitions either. It was more of how a series page could be structured with the column on the left and those episode cards in Vue more or less ;) 

The current Vue way has some limitations in how we can see if a season has been watched already and how to quickly get to other episodes.
